test/attr/attr_downgrade_exhaustivity.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %target-typecheck-verify-swift -swift-version 3 %s
+
+enum Runcible {
+  case spoon
+  case hat
+  @_downgrade_exhaustivity_check
```

Comment:
How much harder would it be to make the attribute versioned, so we can make these into errors in Swift 4 mode?